cdk/examples/generative_ai_service/lib/gen_ai_service_stack.py

```diff
@@ -56,7 +56,11 @@ def __init__(
         fargate_task_image = ApplicationLoadBalancedTaskImageOptions(
             container_name=self.stack_props.container_name,
             # build container image from local folder
-            image=ContainerImage.from_asset("web-app", platform=Platform.LINUX_AMD64),
+            # image=ContainerImage.from_asset("web-app", platform=Platform.LINUX_AMD64),
```

Comment:
nit: remove lines 58-59 if no longer building image from source

Reply:
seems to be a temp measure for re:Invent